index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>My_portfolio</title>
+</head>
+<body>
+    <header>
+            <h1>Kandy</h1>
+            <nav>
+                <i class="fa fa-bars"></i>
+            </nav>
+    </header>
+
+    <section class="clm-1">
+        <div class="clm-pres">
+            <h2>I'm Kandy,<br/> Glad to see you!</h2>
+            <h6>I’m a software developer! I can help you build a product , 
```

Comment:
- [ ] According to HTML semantic tag and HTML best practice in this project the text `I’m a software developer! I can help you build a product....` is not a heading `<h2>` rather is a paragraph `</p>`. Kindly put it in a `<p>` tag. I know you can do it :clap: :clap: